### PR TITLE
add sttia

### PR DESCRIPTION
**Pull request by riley-stride**
### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] Ready -- A liquidity pool has been created. The pool ID is: ______
- [x] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [ ] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -2508,6 +2508,12 @@
       "base_denom": "stadydx",
       "path": "transfer/channel-326/stadydx",
       "osmosis_verified": true
+    },
+    {
+      "chain_name": "stride",
+      "base_denom": "stutia",
+      "path": "transfer/channel-326/stutia",
+      "osmosis_verified": true
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:2516]: Please add "osmosis_unlisted": true

**Comment by JeremyParish69** [comment]
Please see checklist. Is there a pool ID?

File: osmosis-1/osmosis.zone_assets.json (hunk 1 above)
Comment (riley-stride) [reply osmosis-1/osmosis.zone_assets.json:2516]: done

**Comment by riley-stride** [comment]
> Please see checklist. Is there a pool ID?

No Pool ID yet. Updated the checklist.

**Comment by riley-stride** [comment]
@JeremyParish69 can we merge this please?

**Pull request merged**